gadgets/Makefile

```diff
@@ -101,19 +104,43 @@ clean:
 		sudo -E $(IG) image remove $(GADGET_REPOSITORY)/$$GADGET:$(GADGET_TAG); \
 	done
 
-.PHONY:
-test: build
+.PHONY: 
+test-unit: build
+	IG_VERIFY_IMAGE=$(IG_VERIFY_IMAGE) \
+	GADGET_TAG=$(GADGET_TAG) \
+	GADGET_REPOSITORY=$(GADGET_REPOSITORY) \
+	go test -v -exec 'sudo -E' ./.../$(UNIT_TEST_DIR)/...
+
+
+.PHONY: 
+%/test-unit: build
```

Comment:
%/test-unit: %

To avoid compiling all gadgets when running unit tests for a single one.

(same in other similar cases)